rules/windows/file/file_event/file_event_win_create_non_existent_dlls.yml

```
@@ -1,22 +1,30 @@
-title: Creation Of Non-Existent System DLL
+title: Creation Of Non-Existent System DLL - Potential Preparation of Phantom DLL Hijacking
```

The potential portion would go in the description. The title is accurate enough.
```suggestion
title: Creation Of Non-Existent System DLL
```